crates/services/executor/src/executor.rs

```diff
@@ -665,6 +765,11 @@ where
                 .take(remaining_tx_count as usize)
                 .peekable();
         }
+        if !status.is_empty() {
+            if let Err(e) = self.preconfirmation_sender.send(status).await {
```

Comment:
Can we add some executor tests to show that this indeed sends the expected txs through the channel?

Reply:
++

Reply:
Added.